modules/java-modules/http/src/main/java/org/syslog_ng/http/HTTPDestination.java

```diff
@@ -96,15 +95,10 @@ public boolean send(String message) {
                 osw.write(message);
                 osw.close();
             } catch (IOException e) {
-                logger.error("error in writing message.");
-                return false;
+		logger.error("error in writing message.");
+		return false;
```

Comment:
If the whitespace changes are intentional changes then it is better to move to a separate commit message.